packages/table/src/common/_loading.scss

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
```

Comment:
I suppose the loading stuff living here is okay. Maybe a separate subdirectory should be considered as well.

As far as the SASS, I think i would prefer to make the rules below a mixin and then have bp-table-cell and bp-table-header mix them in in their respective .scss files.